src/requests/getForecast.js

```diff
@@ -0,0 +1,22 @@
+// import axios from "axios";
```

Comment:
Instead of defining `getForecast` in `App.js` as youre doing now, why not use this file and keep things separate? If this function is not getting separated into its own file, theres not reason to have all the code duplicated and commented out like this